neurons/validators/scraper_validator.py

```diff
@@ -172,86 +172,97 @@ async def run_task_and_score(
-        end_date = date_filter.end_date.strftime("%Y-%m-%dT%H:%M:%SZ")
-
-        axons = [self.neuron.metagraph.axons[uid] for uid in uids]
-        synapse = ScraperStreamingSynapse(
-            prompt=prompt,
-            model=self.model,
-            seed=self.seed,
-            # is_intro_text=is_intro_text,
-            start_date=start_date,
-            end_date=end_date,
-            date_filter_type=date_filter.date_filter_type.value,
-            tools=tools,
-            language=language,
-            region=region,
-            google_date_filter=google_date_filter,
-            response_order=response_order.value,
-        )
+            start_date = date_filter.start_date.strftime("%Y-%m-%dT%H:%M:%SZ")
+            end_date = date_filter.end_date.strftime("%Y-%m-%dT%H:%M:%SZ")
+            axons = [self.neuron.metagraph.axons[uid] for uid in uids]
+            synapse = ScraperStreamingSynapse(
+                prompt=prompt,
+                model=self.model,
+                seed=self.seed,
+                # is_intro_text=is_intro_text,
+                start_date=start_date,
+                end_date=end_date,
+                date_filter_type=date_filter.date_filter_type.value,
+                tools=tools,
+                language=language,
+                region=region,
+                google_date_filter=google_date_filter,
+                response_order=response_order,
+            )
 
-        # Make calls in groups to avoid AioHTTP 100 connections limit
-        axon_group_1 = axons[:80]
-        axon_group_2 = axons[80:160]
-        axon_group_3 = axons[160:]
-
-        async_response_groups = await asyncio.gather(
-            *[
-                asyncio.create_task(
-                    self.neuron.dendrite1.forward(
-                        axons=axon_group_1,
-                        synapse=synapse,
-                        timeout=self.timeout,
-                        streaming=self.streaming,
-                        deserialize=False,
-                    )
-                ),
-                asyncio.create_task(
-                    self.neuron.dendrite2.forward(
-                        axons=axon_group_2,
-                        synapse=synapse,
-                        timeout=self.timeout,
-                        streaming=self.streaming,
-                        deserialize=False,
-                    )
-                ),
-                asyncio.create_task(
-                    self.neuron.dendrite3.forward(
-                        axons=axon_group_3,
-                        synapse=synapse,
-                        timeout=self.timeout,
-                        streaming=self.streaming,
-                        deserialize=False,
-                    )
-                ),
-            ]
-        )
+            # Make calls to the network with the prompt.
+            async_responses = await self.neuron.dendrite.forward(
```

Comment:
@Chkhikvadze @okradze is this ok? I think I've seen this sneak in as a bug before!